src/rippled/lib/utils.js

```diff
@@ -96,8 +96,6 @@ function RippledError(message, code) {
   this.code = code
 }
 
-require('util').inherits(RippledError, Error)
```

Comment:
Why was this removed?

Reply:
It was complaining when I ran `npm start`

Reply:
I also don't really understand what it's doing

Reply:
It adds all properties of Error to RippledError.

Reply:
Wouldn't it be easier to do that with normal inheritance?

Reply:
That works I just want to make sure the error properties/methods are there.

Reply:
I don't really know how to add this back in - some help would be appreciated.